xml2cards.py

```diff
@@ -34,7 +34,26 @@ def get_type_info(item_type):
     return {'name': 'Unknown type', 'icon': 'cross-mark', 'color': 'black'}
 
 
-def get_item_properties():
+def get_item_properties(sortedItems=False):
```

Comment:
It's fine that this function always returns the properly sorted array. You don't need the sortedItems parameter, and you can remove the old dictionary.